src/scrabble.js

```diff
@@ -1,5 +1,98 @@
-function scrabble() {
-  // write code here
+// make a function that will have an input of a string
+// this function needs to return a number so it will require a counter
+// this function will have to go to each letter of the input and update this counter to a certain amount
+// this function will require to see if the input has any of the following letters regardless of its case
+
+function scrabble(input) {
+  let counter = 0
+  if (input === null || input.length === 0) {
+    return counter
+  }
+  for (let i = 0; i < input.length; i++) {
+    const eachLetter = input[i]
+    if (
+      eachLetter === `a` ||
```

Comment:
Try to use [.toLowerCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase) to halve your checks, as well as having a data structure to contain your letter scores. This way you should be able to reduce your lines of code as well as reducing your total copy-pasting